stave/models.py

```diff
@@ -702,6 +702,30 @@ class Meta:
         ]
         ordering = ["order_key"]
 
+    def user_by_group_and_role(self, role_group: str, role: str) -> User | None:
+        """
+        Returns the User for the given role group and role for the game.
+        """
+        for rga in self.get_crew_assignments_by_role_group().values():
+            if rga.role_group.name != role_group:
+                continue
+            for ca in rga.effective_crew():
+                if ca.role.name == role:
+                    return ca.user
+        return None
```

Comment:
What about something like this, to avoid the need to iterate through RoleGroupCrewAssignments?

(Aside, sorry about how weird some of the schema names are. You can see where I did some revisions!)

```suggestion
        rga = self.role_groups.filter(role_group__name=role_group).first()
        if rga:
           for ca in rga.effective_crew():
                if ca.role.name == role:
                    return ca.user
        return None
```